packages/braintree-integration/src/braintree-fastlane/braintree-fastlane-payment-strategy.ts

```diff
@@ -90,6 +101,7 @@ export default class BraintreeFastlanePaymentStrategy implements PaymentStrategy
         this.initializeCardComponent();
 
         braintreefastlane.onInit((container) => this.renderBraintreeAXOComponent(container));
+        braintreefastlane?.onChange(() => this.handleBraintreeStoredInstrumentChange(methodId));
```

Comment:
onChange will always be available here, you don't need to add the `?` operator.

Reply:
As i see this, when we will not enable fastlane there will be no onChange callback for sure and we try to force call onChange and it will cause error

Reply:
After your updates it makes sense, thank you